### ISSUE TITLE
Multi-word city name search bug

### ISSUE THREAD

**Issue by Seydulla** [ResulShamuhammedov/howabot#2]
When searching for the weather in a city with a two-word name (e.g. 'New York'), the slackbot is only able to recognize the first word and returns inaccurate results.

**Comment by resulshm** [comment]
Thanks for your feedback, I really appreciate it. 
In the shortest possible time, I will fix it

**Comment by jepbarasgarov** [comment]
@resulshm I would like to ask if this issue has been resolved

**Comment by Seydulla** [comment]
Yes i'm curious too. Because of this issue im living in Bejing, Dubai

**Comment by resulshm** [comment]
Fill free to fork and open the PR, guys🙂
